.ci/openshift-ci/cluster/deploy_webhook.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script builds the kata-webhook and deploys it in the test cluster.
+#
+# You should export the KATA_RUNTIME variable with the runtimeclass name
+# configured in your cluster in case it is not the default "kata".
+#
+set -e
+
+script_dir="$(dirname $0)"
+webhook_dir="${script_dir}/../../../kata-webhook"
+hello_pod_name="hello-openshift"
+kata_runtimeclass_name=${KATA_RUNTIME:-kata}
+
+source "${script_dir}/../../lib.sh"
+
+pushd "${webhook_dir}" >/dev/null
+# Build and deploy the webhook
+#
+info "Builds the kata-webhook"
+./create-certs.sh
+info "Deploys the kata-webhook"
+oc apply -f deploy/
+# Wait until it is not available.
+oc wait deployment/pod-annotate-webhook --for condition=Available --timeout 60s
+
+# Check the web-hook is working as expected.
+#
+[ oc get pod/${hello_pod_name} &>/dev/null ] && \
```

Comment:
i didn't get this one, does the pod might exist already?
also i'd check the success of the previous command
fine with me other than that, thanks!

Reply:
> i didn't get this one, does the pod might exist already?

It's unlikely, but you never know :)

> also i'd check the success of the previous command (making sure the waiting didn't timeout)

On failure of previous command the script will exit because it is `set -e`. However I will add a check and gentle fail message.